pulsar-broker/src/main/java/org/apache/pulsar/broker/service/AbstractDispatcherMultipleConsumers.java

```diff
@@ -239,5 +239,24 @@ private int getFirstConsumerIndexOfPriority(int targetPriority) {
 
     private static final Logger log = LoggerFactory.getLogger(PersistentStickyKeyDispatcherMultipleConsumers.class);
 
+    public StickyKeyConsumerSelector createConsumerSelectorInstance(ServiceConfiguration conf) {
+        try {
+            Class<?> consumerSelectorClass = Class.forName(conf.getKeySharedConsumerSelectorStrategy());
+            Object consumerSelectorClassInstance = consumerSelectorClass.getDeclaredConstructor().newInstance();
+            if (consumerSelectorClassInstance instanceof StickyKeyConsumerSelector) {
+                return (StickyKeyConsumerSelector) consumerSelectorClassInstance;
+            } else {
+                log.error("create key shared consumer selector strategy failed."
```

Comment:
We cannot fallback to another algorithm otherwise the system will break as all the assumptions in the application won't be guaranteed.

Let's throw a bad error here and in other failure cases